### PR TITLE
column in a single place

### PR DESCRIPTION
**Pull request by adlerhurst**

### CODE REVIEW

File: internal/query/projection/org.go [hunk 1]
```diff
@@ -58,17 +58,31 @@ func (p *OrgProjection) reducers() []handler.AggregateReducer {
...
+	OrgColumnChangeDate    = "change_date"
+	OrgColumnResourceOwner = "resource_owner"
+	OrgColumnState         = "org_state"
+	OrgColumnSequence      = "sequence"
+	OrgColumnName          = "name"
+	OrgColumnDomain        = "primary_domain"
 )
 
+func (c OrgColumn) ColumnName() string {
+	switch c {
```
Comment (fgerschwiler) [inline internal/query/projection/org.go:75]: I think this isn't used anymore

File: internal/query/projection/org.go [hunk 2]
```diff
@@ -58,17 +58,31 @@ func (p *OrgProjection) reducers() []handler.AggregateReducer {
...
+
+	default:
+		return ""
+	}
+}
+
+func (c OrgColumn) FullColumnName() string {
+	return OrgProjectionTable + "." + c.ColumnName()
```
Comment (fgerschwiler) [inline internal/query/projection/org.go:83]: I think this isn't used anymore

File: internal/query/org.go [hunk 3]
```diff
@@ -12,17 +12,60 @@ import (
...
-			OrgColumnID.toFullColumnName(),
-			OrgColumnCreationDate.toFullColumnName(),
-			OrgColumnChangeDate.toFullColumnName(),
-			OrgColumnResourceOwner.toFullColumnName(),
-			OrgColumnState.toFullColumnName(),
-			OrgColumnSequence.toFullColumnName(),
-			OrgColumnName.toFullColumnName(),
-			OrgColumnDomain.toFullColumnName()).
-			From(projection.OrgProjectionTable).PlaceholderFormat(sq.Dollar),
+			OrgColumnID.sql(),
```
Comment (fgerschwiler) [inline internal/query/org.go:59]: I would preferre to name this function different than sql(), for me its not clear what it does

**Review by fgerschwiler — APPROVED** [review]

File: internal/query/org.go (hunk 3 above)
Comment (adlerhurst) [reply internal/query/org.go]: renamed to identifier

File: internal/query/projection/org.go (hunk 1 above)
Comment (adlerhurst) [reply internal/query/projection/org.go]: you're right. Code is removed

File: internal/query/projection/org.go (hunk 2 above)
Comment (adlerhurst) [reply internal/query/projection/org.go]: you're right. Code is removed

**Review by fgerschwiler — APPROVED** [review]

**Pull request merged**